res/css/views/rooms/_EventTile.pcss

```diff
@@ -131,11 +131,6 @@ $left-gutter: 64px;
         }
     }
 
-    &.mx_EventTile_highlight,
```

Comment:
Red event highlighting is used to highlight any event that matches a push rule with a `highlight` tweak - eg @mentions, display name mentions, and keywords mentions - so this can't be removed without addressing those cases.

Reply:
Added this back in andchanged line 666 of EventTile.tsx to not highlight reply text instead